### DISCUSSION TITLE
哥们，这东西你们更新了有点块啊😧

### DISCUSSION THREAD

**Discussion by Sjshi763** [wushuo894/ani-rss#28] [General]
what can ｉ say

**Comment by Sjshi763** [comment]
不过也许你们可以把电影也改成只要标题（下载）

**Comment by wushuo894** [comment]
一天发10版是跟你闹着玩的吗😁

**Reply by Sjshi763** [reply]
我勒个豆，马上卷5块

**Reply by Sjshi763** [reply]
不感觉稳定性还有提高

**Reply by wushuo894** [reply]
> 我勒个豆，马上卷5块

感谢老爷，老爷万事如意

**Reply by Sjshi763** [reply]
不过还是要改善稳定性

**Reply by Sjshi763** [reply]
byd下番下完了我直接找不到了

**Reply by Sjshi763** [reply]
必须删qb的资源重来

**Reply by wushuo894** [reply]
> byd下番下完了我直接找不到了

可以详细描述吗

**Reply by wushuo894** [reply]
> byd下番下完了我直接找不到了

有个自动删除已做完种任务的开关

**Reply by Sjshi763** [reply]
就是集体导入了一波 为美好世界献上祝福3 结果没干上去 那个还不是我删了，并且我上一个删qb资源是我眼花了，是另一个番的

**Reply by Sjshi763** [reply]
问题我在下载地址也找不到

**Reply by Sjshi763** [reply]
byd找了一下好像更本没下

**Reply by wushuo894** [reply]
> byd找了一下好像更本没下

看下日志有没有添加下载

**Reply by Sjshi763** [reply]
没有

**Reply by Sjshi763** [reply]
好了，已经整个重下了

**Reply by Sjshi763** [reply]
成功了，不过估计不是你的问题我的nas不带docker

**Reply by wushuo894** [reply]
> 成功了，不过估计不是你的问题我的nas不带docker

💕

**Reply by Sjshi763** [reply]
豪勇🤓🤓🤓🤓☝️☝️☝️☝️🫡🫡🫡🫡😋😋😋😋😋🤔🤔🤔不过电影要是可以直接下而不是还有制作组就更好了

**Reply by Sjshi763** [reply]
![Uploading IMG_1592.png…]()耗用

**Reply by wushuo894** [reply]
> 豪勇🤓🤓🤓🤓☝️☝️☝️☝️🫡🫡🫡🫡😋😋😋😋😋🤔🤔🤔不过电影要是可以直接下而不是还有制作组就更好了

因为每个制作组都有不同的压制方式

编码不通，码率和分辨率也不一样，有的也只做繁体，有的只做简体。有人喜欢繁简都有的

**Reply by Sjshi763** [reply]
我的意思是下载后自动改名

**Reply by wushuo894** [reply]
> 我的意思是下载后自动改名

这个啊，很多字幕组会在ova夹带私货，或者上传复杂结构

所以现在还没有更好的方案，但是即使不重命名，emby在一个文件夹下只有一个视频的情况下是使用文件夹的名字进行刮削的，所以不会影响

**Reply by Sjshi763** [reply]
用video station kodi infuse的我，what can I say？

**Reply by Sjshi763** [reply]
对了tmdb是还没做吗？没找到API的地方

**Reply by wushuo894** [reply]
> 对了tmdb是还没做吗？没找到API的地方

用的是mikan和番组计划刮削的

**Reply by wushuo894** [reply]
> 用video station kodi infuse的我，what can I say？

在7.2.2群晖已经把他砍了，赶紧找平替吧

**Reply by Sjshi763** [reply]
已经开始了，但是只要我不升就可以

**Reply by Sjshi763** [reply]
并且我是用quickconnect的，好像外网不能直接用emby

**Comment by Sjshi763** [comment]
眨眼之间又是两个版本，开始怀疑下周回来可不可以正常更新了

**Reply by Sjshi763** [reply]
别版本跨度太大紫菜我

**Comment by Sjshi763** [comment]
又三次

**Comment by Sjshi763** [comment]
一次一次又一次
